xknx/knxip/knxip_enum.py

```diff
@@ -167,3 +167,26 @@ class SearchRequestParameterType(Enum):
     SELECT_BY_SERVICE = 0x03
     # The Client shall include this SRP to indicate that it is interested in the listed DIBs
     REQUEST_DIBS = 0x04
+
+
+class TunnellingFeatureType(Enum):
+    """Enum class for KNX/IP Tunnel Features services."""
+
+    # Invalid placeholder
+    INVALID = 0x00
```

Comment:
is this used in the KNX specs too?

Reply:
It's not in the specs, I added it to have a default argument for init functions. I can remove it and make the argument required if that's preferred?

Reply:
Making it required doesn't really work with the current implementation of KNXIPFrame. See https://github.com/XKNX/xknx/blob/bf69e38792d61ec2e84c8ec175c3a61a3f20c4c6/xknx/knxip/knxip.py#L122-L123
This would require a quite big refactoring - like done for APCI (see https://github.com/XKNX/xknx/pull/1154) but that is out of scope.

You can leave it there and maybe add a check to raise if it was encountered in `to_knx` or `from_knx` (just like any other, not listed enum-item would do). 
Or just use a different `TunnellingFeatureType` as default (we do this in other KNXIPBody arguments too (`sequence_counter`, `communication_channel_id`, etc.) - not beautiful, but works fine 😬

Reply:
I will change the default.